keanu-project/src/test/java/io/improbable/keanu/util/csv/pojo/bycolumn/CsvReaderByColumnTest.java

```diff
@@ -0,0 +1,166 @@
+package io.improbable.keanu.util.csv.pojo.bycolumn;
+
+import io.improbable.keanu.tensor.bool.BooleanTensor;
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
+import io.improbable.keanu.tensor.intgr.IntegerTensor;
+import io.improbable.keanu.util.csv.ReadCsv;
+import io.improbable.keanu.util.csv.pojo.CsvProperty;
+import org.junit.Test;
+
+import static org.apache.commons.lang3.ArrayUtils.toPrimitive;
+import static org.junit.Assert.assertArrayEquals;
+
+public class CsvReaderByColumnTest {
+
+    String csv = "a,b,c\n0,2.5,t\n1,3,true\n-3,4e3,0";
```

Comment:
nit: can we format this to look like a preview of the csv file?

```
    String csv = "a,b,c\n" +
                 "0,2.5,t\n" +
                 "1,3,true\n" +
                 "-3,4e3,0";
```

Reply:
Good call.